server/src/lib.rs

```diff
@@ -330,7 +359,7 @@ async fn index(ci: ConnectionInfo) -> Result<Json<Vec<url::Url>>, UrlGenerationE
 
     result.extend(build_url(&ci, "/"));
     result.extend(build_url(&ci, "/openapi.json"));
-    result.extend(build_url(&ci, "/openapi/"));
+    result.extend(build_url(&ci, "/openapi.json"));
```

Comment:
question: seems to duplicate lines and remove the swagger UI url?

Reply:
That's a mistake. Thanks !